test/kefir-test-utils.spec.js

```diff
@@ -234,7 +234,7 @@ describe('kefir-test-utils', () => {
       }
 
       withFakeTime(tick => {
-        log = watchWithTime(obs)
+        log = watchWithTime(obs).log
```

Comment:
```suggestion
        ({log} = watchWithTime(obs))
```

Reply:
I had to revert this change, this is what prettier formats this line like: `;({log} = watchWithTime(obs))`